lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/rx/impl/RxThreadPoolStage.java

```diff
@@ -168,7 +168,7 @@ public void run() {
   /**
    * Closes the stage.
    *
-   * @return Exception
```

Comment:
I make that according to the function definition `public void close() throws Exception {`.
It's `void` function but has `throws`. But, as you pointed here, we had better remove that line.
I agree with you.